File: Applications.GenuisReader/Applications.GenuisReader/ClientApp/src/app/models/tag.ts
export class Tag {
    public tagId: number;
    public label: String;
    public description: String;
    public isSensitive: boolean;
    constructor(id: number, label: String, description: String, isSensitive: boolean) {
        this.tagId = id;
        this.label = label;
        this.description = description;
        this.isSensitive = isSensitive;
    }
}

File: Applications.GenuisReader/GeniusReader.Domain/EFModel/Tag.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GeniusReader.Domain.EFModel
{
    [Table("Tag", Schema = "dbo")]
    public class Tag
    {
        public int TagId { get; set; }
        public string Label { get; set; }
        public string Description { get; set; }
        public bool? IsSensitive { get; set; }
        public List<Chapter> ChaptersTagIsUsedIn { get; set; }
        public List<Series> SeriesTagIsUsedIn { get; set; }
    }
}
